### PR TITLE
fix(core): Fix improvements page track order ecommerce

### PR DESCRIPTION
**Pull request by JoanSerna**

### CODE REVIEW

File: libs/models/order-ecommerce-request.ts [hunk 1]
```diff
@@ -13,5 +13,5 @@ export class OrderEcommerceRequest {
   orders!: {
     quantity: number;
     product: ProductEcommerceEntity;
-  };
+  }[];
```
Comment (FranzLuepke) [inline libs/models/order-ecommerce-request.ts:16]: What is the purpose of this?

Comment (JoanSerna) [reply libs/models/order-ecommerce-request.ts:16]: The model was wrong, it’s array of orders

File: apps/guest/src/product/pages/order-detail-ecommerce/order-detail-ecommerce.component.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
+@import 'styles/variables';
```
Comment (FranzLuepke) [inline apps/guest/src/product/pages/order-detail-ecommerce/order-detail-ecommerce.component.scss:1]: Shouldn't this just work?
``` javascript
@import 'variables';
```

File: apps/guest/src/product/pages/order-detail-ecommerce/order-detail-ecommerce.component.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+.detail {
+  &-id {
+    font-family: $family;
+    font-size: 14.2px;
```
Comment (FranzLuepke) [inline apps/guest/src/product/pages/order-detail-ecommerce/order-detail-ecommerce.component.scss:6]: Shouldn't `mixin` be used?

Something like:
``` typescript
@import 'mixin';
...
@include text($text-3, 20px, 500);
```

File: apps/guest/src/product/pages/order-detail-ecommerce/order-detail-ecommerce.component.scss [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    line-height: 1.41;
+    letter-spacing: 0.25px;
+    text-align: center;
+    color: rgba(0, 0, 0, 0.87);
+    margin: 30px 0 15px;
+  }
+
+  &-total {
+    font-family: $family;
+    font-size: 20.2px;
```
Comment (FranzLuepke) [inline apps/guest/src/product/pages/order-detail-ecommerce/order-detail-ecommerce.component.scss:16]: Same from `mixin`.

File: apps/guest/src/product/pages/order-detail-ecommerce/order-detail-ecommerce.component.scss [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    text-align: center;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: 0.25px;
+  }
+
+  &-comment {
+    font-family: $family;
+    font-size: 14.2px;
```
Comment (FranzLuepke) [inline apps/guest/src/product/pages/order-detail-ecommerce/order-detail-ecommerce.component.scss:28]: Same from `mixin`.

File: apps/guest/src/product/pages/order/order.component.scss [hunk 6]
```diff
@@ -18,8 +19,50 @@
...
+
+    &__dates {
+      display: flex;
+      align-items: center;
+    }
+  }
+
+  &__title {
+    font-family: $family;
+    font-size: 20.2px;
```
Comment (FranzLuepke) [inline apps/guest/src/product/pages/order/order.component.scss:46]: Same from `mixin`.

File: apps/guest/src/product/pages/order/order.component.scss [hunk 7]
```diff
@@ -18,8 +19,50 @@
...
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: 0.25px;
+    color: rgba(0, 0, 0, 0.87);
+    margin: 0;
+  }
+
+  &__text {
+    font-family: Raleway, sans-serif;
+    font-size: 12.2px;
```
Comment (FranzLuepke) [inline apps/guest/src/product/pages/order/order.component.scss:58]: Same from `mixin`.

**Review by FranzLuepke — CHANGES_REQUESTED** [review]
I left some comments.

File: libs/models/order-ecommerce-request.ts (hunk 1 above)
Comment (FranzLuepke) [reply libs/models/order-ecommerce-request.ts:16]: Oh! I see. Alright.

**Review by FranzLuepke — APPROVED** [review]

**Pull request merged**